### PR TITLE
Child dashboard

### PR DESCRIPTION
**Pull request by ghost**
# Description

As a child, I would like to view current the activity, so that I am able to participate in the weekly sessions

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Rendering Test

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by ghost** [comment]
[[Child] - View Current Activity](https://trello.com/c/yuJUJXrS/32-child-view-current-activity)

**Review by wSedlacek — APPROVED** [review]
Looks good to me.

File: src/app/pages/child-dashboard/child-dashboard.page.tsx [hunk 1]
```diff
@@ -1,11 +1,81 @@
...
+    }, [request]);
+
+    if (!response?.me)
+        return (
+            <section className={classes.loading}>
+                <CircularProgress size={56} />
+            </section>
+        );
+
+    console.log(response?.me);
```
Comment (EricBanker12) [inline src/app/pages/child-dashboard/child-dashboard.page.tsx]: this console.log should be removed

**Review by EricBanker12 — COMMENTED** [review]
Testing and Manual usage seem to work. A missed console log in child-dashboard page should be removed.

**Pull request merged**